etl/steps/data/garden/trade/2025-09-22/historic_trade.meta.yml

```diff
@@ -0,0 +1,26 @@
+# NOTE: To learn more about the fields, hover over their names.
+definitions:
+  common:
+    presentation:
+      topic_tags:
+        - Trade & Globalization
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+dataset:
+  update_period_days: 709
+
+tables:
+  historic_trade:
+    variables:
+      volume_index:
+        title: Trade volume index
+        unit: index (1913 = 100)
+        description_short: The average of export and import volumes, reconstructed from historical sources and standardized to modern definitions, expressed as an index with 1913 = 100.
+        description_key:
+          - International trade has long been central to economic development. To trace its evolution, researchers compiled historical trade statistics from hundreds of sources, covering nearly all countries and colonies from 1800 onward. While the UK began reporting as early as 1696, most systematic records only appeared in the 19th century, and each country used different methods.
```

Comment:
Up to you but it could be a nice spot to mention the researchers by name and link out to their paper